client/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:10.15-alpine
+WORKDIR /usr/src/app
+COPY package.json yarn.lock ./
+RUN yarn install
+COPY . .
```

Comment:
did you think of adding a `.dockerignore` file in order not to copy everything into the image (including nodes_modules) ?

Reply:
I'll add it!

Reply:
:+1: